packages/testing/config/jest/api/apiBabelConfig.js

```diff
@@ -0,0 +1,21 @@
+const {
+  getApiSideDefaultBabelConfig,
+  getApiSideBabelPresets,
+  getApiSideBabelPlugins,
+} = require('@redwoodjs/internal')
+
+// Since configFile and babelrc is already passed a level up, cleaning up these keys here.
+// Ref: packages/testing/config/jest/api/index.js
+const {
+  configFile: _c,
```

Comment:
I appreciate what you've done here, and it makes sense. I think however the problem is that we're using `configFile` in the api defaultBabelConfig in the first place, I'm going to try changing it to extends to be consistent with web. To be clear, the problem isn't with your code here, but the way I set up the default config before.

Like you noted, configFile can only be specified once, which means that if we don't change it to extends, if the user has custom babel config for their apiside, it won't be picked up.